start-up/moderation-tutorial.md

```diff
@@ -1,54 +1,55 @@
+- `notify`: Whether or not the bot will message the user after the command is executed, as shown below
+![An example of a notification from the bot](/.gitbook/assets/NotifyExample.png "Notify Example")
+- `no-infraction`: Whether or not the command will count towards the user's infractions
 
 {% hint style="info" %}
-If both the `notify` and `no-infraction` arguments are true, the target will not be notified in order to prevent harrassment via MoonlightBot
+If both the `notify` and `no-infraction` options are true, the target will not be notified in order to prevent harassment via MoonlightBot
 {% endhint %}
 
-- **`duration`**: The amount of time for which the user will be affected by the command. For more information about this argument, refer to the [arguments page](/start-up/arguments.md#durations)
-- **`remove-roles`**: Exclusive to the mute commands, this argument will remove all of the user's roles except the muterole if true
-- **`role`**: The role to assign/remove in a role command
+- `duration`: The amount of time for which the user will be affected by the effects of the command. You can use any duration ranging from weeks to milliseconds. For example, `1w2d` and `1 week 2 days` will both affect the target of the command for 1 week and 2 days.
+- `remove-roles`: Exclusive to the mute commands, this option will remove all of the user's roles except the muterole if true. This is to ensure the permissions of other roles don't interfere with the effectiveness of the mute.
+- `role`: The role to assign/remove in a role command
 
 ## Moderation Tools
 
 These features help ensure a seamless moderation experience with MoonlightBot
 
 ### Infractions
 
-![](/.gitbook/assets/InfractionsExample.png)
+![An example of the Infractions Summary command](/.gitbook/assets/InfractionsExample.png "Infractions Example")
+
+Infractions help staff members keep track of a user's bad conduct and make informed decisions while moderating. This is a versatile tool with multiple submcommands to assist your moderation needs.
 
-Infractions help staff members organize actions taken while moderating, such as mutes. This is a versatile tool with multiple submcommands to assist your moderation needs. More information about infractions can be found in the [infractions page](/moderation-commands/infractions.md)
+More information about infractions can be found in the [infractions page](/moderation-commands/infractions.md)
 
 ### Logs
 
-![](/.gitbook/assets/LogExample.png)
+![A log example containg a mute infraction](/.gitbook/assets/LogExample.png "Logs Example")
 
 When moderators execute an action such as a warn, MoonlightBot will send a message in the designated logs channel in order to keep track of moderation events. For more information on setting up logs, refer to the logs section of the [main page](/README.md)
```

Comment:
@sks316 What's the exact anchor name for the logs section?

Reply:
That would be `README.md#logging`.

Reply:
```suggestion
When moderators execute an action such as a warn, MoonlightBot will send a message in the designated logs channel in order to keep track of moderation events. For more information on setting up logs, refer to the logs section of the [main page](/README.md#logging)
```